From: Aleix Pol <aleixpol@kde.org>
Date: Wed, 18 Jan 2017 12:15:00 +0100
Subject: Fix build

frameworks/konsole/src/Session.cpp:839:5: warning: 'not' is not defined,
evaluates to 0 [-Wundef]
---
 src/Session.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Session.cpp b/src/Session.cpp
index d63c8a0c..482ee0b6 100644
--- a/src/Session.cpp
+++ b/src/Session.cpp
@@ -836,7 +836,7 @@ void Session::sendTextToTerminal(const QString& text, const QChar& eol) const
 // Only D-Bus calls this function (via SendText or runCommand)
 void Session::sendText(const QString& text) const
 {
-#if not defined(REMOVE_SENDTEXT_RUNCOMMAND_DBUS_METHODS)
+#if !defined(REMOVE_SENDTEXT_RUNCOMMAND_DBUS_METHODS)
     if (show_disallow_certain_dbus_methods_message) {
 
         KNotification::event(KNotification::Warning, "Konsole D-Bus Warning",
